distribution/scripts/ballerina/data_mapping/main.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/http;
+
+configurable string epKeyPath = ?;
+configurable string epKeyPassword = ?;
+
+type Item record {|
+    string symbol;
+    string buyerID;
+    float price;
+    int volume;
+|};
+
+type OrderItem record {|
+    string symbol;
+    string purchaser;
+    float unitPrice;
+    int quantity;
+    float totalCost;
+|};
+
+listener http:Listener securedEP = new (9090,
+    secureSocket = {
+        key: {
+            path: epKeyPath,
+            password: epKeyPassword
+        }
+    }
+);
+
+final http:Client nettyEP = check new ("https://netty:8688",
+    secureSocket = {
+        cert: {
+            path: epKeyPath,
+            password: epKeyPassword
+        },
+        verifyHostName: false
+    }
+);
+
+service /dataMapping on securedEP {
+    resource function post .(@http:Payload json data) returns json|error? {
```

Comment:
Any specific reason to avoid payload-binding here and try to bind it inside the resource function? Internally we use the same APIs like `fromJsonWithType`